### PR TITLE
TW 347 RBAC on the Private API Network

### PR DESCRIPTION
**Pull request by HeatherStenson**
- Adds information about new role and workflow
- Reorganizes Private API Network page for readability/flow
- Formats instructions as numbered lists

Still to do: Update some screenshots

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 1]
```diff
@@ -34,179 +34,266 @@ contextual_links:
...
+<img alt="API page in private network" src="https://assets.postman.com/postman-docs/private-network-api-view-v9.1.jpg"/>
 
-Select one or more APIs you want to add to the network at one time. Create a folder to add the APIs to, if required. Select **Add API**.
+To review version-level details, select **View** next to the API version. If you have an Editor role for the API, you can edit the schema or make changes to the API directly.
 
-<img alt="Add API Modal" src="https://assets.postman.com/postman-docs/add-to-private-api-network-v9.jpg" width="400px"/>
+ <img alt="View an API version" src="https://assets.postman.com/postman-docs/private-api-network-view-version-v9.jpg" width="500px"/>
 
-> The APIs that you add to the Private API Network reflect the latest state of the API in your team workspace. In other words, changes made to the API in the workspace will be reflected in the network in real time.
+To watch the API and get notified about any changes, select **Watch**. To learn more about watch notifications, see [Watching an API](/docs/designing-and-developing-your-api/managing-apis/#watching-apis).
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:79]:
```suggestion
To watch the API and get notified about any changes, select **Watch** in the upper right corner. To learn more about watch notifications, see [Watching an API](/docs/designing-and-developing-your-api/managing-apis/#watching-apis).
```

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 2]
```diff
@@ -34,179 +34,266 @@ contextual_links:
...
 
-Learn more about [editing and versioning APIs](/docs/designing-and-developing-your-api/versioning-an-api/).
+To use this approval process for your team, you need to complete two steps first:
 
-#### Security
+1. [Assign the API Network Manager role to a user](#assign-the-api-network-manager-role)
+1. [Turn on the approval process in Team Settings](#turn-on-the-approval-process)
 
-Private APIs are only visible to logged in users who are a part of your Postman team. They're not discoverable or accessible to anyone who is not a part of your team.
+Once these steps are complete, users with an Editor role for an API will need to [request to add the API](#editor-requesting-to-add-an-api) to the Private API Network. The API Network Manager will be able to [review requests](#api-network-manager-reviewing-requests-to-add-apis) to add an API to the Private API Network, [add APIs](adding-your-apis), and [create and edit folders](#organizing-with-folders).
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:92]:
```suggestion
Once these steps are complete, users with an Editor role for an API will need to [request to add the API](#editor-requesting-to-add-an-api) to the Private API Network. The API Network Manager will be able to [review requests](#api-network-manager-reviewing-requests-to-add-apis) to add an API to the Private API Network, [add APIs](#adding-your-apis), and [create and edit folders](#organizing-with-folders).
```

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 3]
```diff
@@ -34,179 +34,266 @@ contextual_links:
...
-To add an API to the network, you must have edit access to the API and the API must be in a team or public workspace. You cannot add an API to the private network unless all team members have at least view access to the API. Learn more about team [roles and permissions](/docs/collaborating-in-postman/roles-and-permissions/).
+An [API Network Manager](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) can:
 
-### Features of Private API Network
+* Add any API to the team's Private API Network
+* Create and edit folders
+* Assign this role to other team members
 
-The Private API Network provides a central directory of all internal APIs in your organization, that teams can check before engineering their own solutions.
+> You must have either [the Super Admin role or the API Network Manager role](docs/administration/managing-your-team/managing-your-team/#managing-roles) to assign this role to a user.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:102]:
```suggestion
> You must have either [the Super Admin role or the API Network Manager role](/docs/administration/managing-your-team/managing-your-team/#managing-roles) to assign this role to a user.
```

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 4]
```diff
@@ -34,179 +34,266 @@ contextual_links:
...
 
-The following lists the key features of the Private API Network that help make discovering Private APIs easier:
+To assign the API Network Manager role:
 
-#### Folder organization
+1. On the Team Settings page, select **Members and groups**.
+1. Select the user you want to assign the API Network Manager role to.
+1. In the **Roles** dropdown list next to their name, select **API Network Manager**, then select **Update Roles**. For more information about assigning team roles to individual users, see [Managing roles](docs/administration/managing-your-team/managing-your-team/#managing-roles).
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:108]:
```suggestion
1. In the **Roles** dropdown list next to their name, select **API Network Manager**, then select **Update Roles**. For more information about assigning team roles to individual users, see [Managing roles](/docs/administration/managing-your-team/managing-your-team/#managing-roles).
```

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 5]
```diff
@@ -34,179 +34,266 @@ contextual_links:
...
+Postman will send an email to new API Network Managers about their updated role.
 
-From the overview page, select **Create Folder** on the right to create a new folder. You can also use **Create folder** to create sub-folders inside a folder.
+### Turn on the approval process
 
-![New folder in Private Network](https://assets.postman.com/postman-docs/add-new-folder-api-network-overview-v9.jpg)
+The approval process enables an API Network Manager to control the process of adding APIs to their team's Private API Network.
 
-Once you've created the folder, you can add APIs to get started. Make your API selection from the dropdown under **Select APIs** to add them to your team's Private API Network. You can select as many APIs you want to add to the Private API Network at one time. Select __Add API__.
+> You must have either [the Super Admin role or the API Network Manager role](docs/administration/managing-your-team/managing-your-team/#managing-roles) to turn on the approval process.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:118]:
```suggestion
> You must have either [the Super Admin role or the API Network Manager role](/docs/administration/managing-your-team/managing-your-team/#managing-roles) to turn on the approval process.
```

File: src/pages/docs/collaborating-in-postman/adding-private-network.md (hunk 4 above)
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:108]:
```suggestion
1. In the **Roles** dropdown list next to their name, select **API Network Manager**, then select **Update Roles**. For more information about assigning team roles to individual users, see [Managing roles](/docs/administration/managing-your-team/managing-your-team/#managing-roles).
```

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 6]
```diff
@@ -34,179 +34,266 @@ contextual_links:
...
+1. Select **Deny**.
+1. Write a note for the API Editor who submitted the request with details about why you are denying the request.
+1. Select **Deny request**. <!-- TODO: screenshot -->
 
-<img alt="Visibility API versions" src="https://assets.postman.com/postman-docs/new-version-visibility-api-v9.jpg"/>
+You can also review a request directly in an API's overview:
 
-You can always decide the visibility of API versions visible to the consumers. To edit an API version, select the version in the **Overview** tab, then select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> > **Edit** in the upper right. You can select the checkbox to make that particular version available to the consumers. Select **Save Changes** when done.
+1. Select **APIs** from the sidebar, then select the API with the pending review.
+1. Details about the request, including comments from the user who submitted it, are on the right side of the overview.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:162]: Option to keep in similar format as list above it
```suggestion
1. View details about the request, including comments from the user who submitted it, on the right side of the overview.
```

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 7]
```diff
@@ -34,179 +34,266 @@ contextual_links:
...
+    <img alt="Add to Private API Network" src="https://assets.postman.com/postman-docs/add-api-to-private-network-v9.1.jpg" width="400px">
 
-<img alt="API landscape governance" src="https://assets.postman.com/postman-docs/api-landscape-governance-v9.jpg"/>
+After you have added the API, it will be visible in your team's [Private API Network](https://go.postman.co/network/private).
+
+The APIs that you add to the Private API Network reflect the latest state of the API in your team workspace. In other words, changes made to the API in the workspace are reflected in the network in real time. Additionally, all API versions that are visible to consumers are available on the Private API Network. Read more about [publishing specific API versions](#publishing-specific-api-versions).
+
+### Publishing specific API versions
+
+You can publish specific versions of your APIs to the Private API Network. You can't publish API versions that aren't visible to consumers.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:203]: I find the second sentence a bit confusing, maybe could rephrase?

File: src/pages/docs/collaborating-in-postman/adding-private-network.md (hunk 5 above)
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:118]:
```suggestion
> You must have either [the Super Admin role or the API Network Manager role](/docs/administration/managing-your-team/managing-your-team/#managing-roles) to turn on the approval process.
```

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 8]
```diff
@@ -68,32 +68,36 @@ You can [assign](/docs/administration/managing-your-team/managing-your-team/) on
 * **Billing**: Manages team plan and payments. Billing roles can be granted by a Super Admin, by a fellow team member with a Billing role, or by an Admin when inviting a new team member. If this is not possible, contact our [Support Center](https://support.postman.com/hc/en-us).
 * **Developer**: Has access to all team resources and workspaces.
 * **Community Manager**: Manages the public visibility of workspaces and team profile ([Professional and Enterprise plans only](https://www.postman.com/pricing)).
+* **API Network Manager**: Manages a team's [Private API Network](/docs/collaborating-in-postman/adding-private-network/), including adding APIs and reviewing requests to add APIs. A Super Admin or another team member with the API Network Manager role can grant this role to a user ([Enterprise plans only](https://www.postman.com/pricing)).
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:71]: Change all these from colons to dashes

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 9]
```diff
@@ -34,179 +34,266 @@ contextual_links:
...
+
+    <img alt="Edit API version" src="https://assets.postman.com/postman-docs/edit-version-api-v9.1.jpg" width="350px"/>
+
+To learn more about the visibility of API versions, see [Versioning APIs](/docs/designing-and-developing-your-api/versioning-an-api/).
+
+### Importing APIs from GitHub
 
 You can make all your existing APIs discoverable on the Private API Network when you import from GitHub. When you create an API, switch over to the **Import** tab to view the options to import from a code repository.
 
 <img alt="GitHub import" src="https://assets.postman.com/postman-docs/github-import-v9.jpg" height="300px"/>
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:226]: (just to bring it a bit more in line with other screenshots)
```suggestion
<img alt="GitHub import" src="https://assets.postman.com/postman-docs/github-import-v9.jpg" width="450px"/>
```

File: src/pages/docs/collaborating-in-postman/adding-private-network.md (hunk 3 above)
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:102]:
```suggestion
> You must have either [the Super Admin role or the API Network Manager role](/docs/administration/managing-your-team/managing-your-team/#managing-roles) to assign this role to a user.
```

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 10]
```diff
@@ -34,179 +34,266 @@ contextual_links:
...
+1. Select __Create Folder__ from the sidebar.
+1. Give the folder a name and a summary.
+1. Select **Save**.
 
-**Add API to Private API Network** will display on the right if you haven't added the API to the private network.
+<img alt="New folder in Private Network" src="https://assets.postman.com/postman-docs/add-new-folder-api-network-left-navigation-v9.jpg" width="350px"/>
+
+To create a new folder from the Private API Network overview page:
+
+1. Select **Create Folder** on the right.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/adding-private-network.md:251]: Might be worth adding this from above here

> 1. Give the folder a name and a summary.
> 1. Select **Save**.

**Review by jkonrath-postman — APPROVED** [review]
LGTM! Don't forget your pending screenshots (I didn't comment on those). Also do a `npm run build` locally and make sure your links are good. I noticed a couple with missing leading slashes.

**Review by claireriley — COMMENTED** [review]
Looks great! Added a few suggestions

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md (hunk 8 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/roles-and-permissions.md]: Updated this for the whole page in [81fdb9e](https://github.com/postmanlabs/postman-docs/pull/4030/commits/81fdb9ec07cf329a24f9c6ff6090dcec2f006d6b)

File: src/pages/docs/collaborating-in-postman/adding-private-network.md [hunk 11]
```diff
@@ -34,179 +34,266 @@ contextual_links:
...
+    * [Importing APIs from GitHub](#importing-apis-from-github)
+* [Managing the Private API Network](#managing-the-private-api-network)
+    * [Organizing with folders](#organizing-with-folders)
+    * [Editing API listings](#editing-api-listings)
+    * [Removing APIs from the Private API Network](#removing-apis-from-the-private-api-network)
+* [Private API Network reports](#private-api-network-reports)
 * [Next steps](#next-steps)
 
-### Adding your APIs
+## Navigating the Private API Network
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/adding-private-network.md:64]: Good call! Updated in [f4aa3af](https://github.com/postmanlabs/postman-docs/pull/4030/commits/f4aa3af8ab5fcbbb0205c6c462a9054dff3a81c4)

File: src/pages/docs/collaborating-in-postman/adding-private-network.md (hunk 7 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/adding-private-network.md]: Agreed! Updated this section in [f4aa3af](https://github.com/postmanlabs/postman-docs/pull/4030/commits/f4aa3af8ab5fcbbb0205c6c462a9054dff3a81c4)

File: src/pages/docs/collaborating-in-postman/adding-private-network.md (hunk 10 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/adding-private-network.md:256]: Thanks, good catch! Updated in [f4aa3af](https://github.com/postmanlabs/postman-docs/pull/4030/commits/f4aa3af8ab5fcbbb0205c6c462a9054dff3a81c4)

File: src/pages/docs/collaborating-in-postman/adding-private-network.md (hunk 4 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/adding-private-network.md]: So many missing! A truly monumental copy-paste error 🙈  Fixed in [3bf591a](https://github.com/postmanlabs/postman-docs/pull/4030/commits/3bf591a303294d467b78da686001f26f2514f5f6)

File: src/pages/docs/collaborating-in-postman/adding-private-network.md (hunk 5 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/adding-private-network.md]: Fixed in [3bf591a](https://github.com/postmanlabs/postman-docs/pull/4030/commits/3bf591a303294d467b78da686001f26f2514f5f6)

File: src/pages/docs/collaborating-in-postman/adding-private-network.md (hunk 3 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/adding-private-network.md]: Fixed in [3bf591a](https://github.com/postmanlabs/postman-docs/pull/4030/commits/3bf591a303294d467b78da686001f26f2514f5f6)

**Pull request merged**